fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -4105,6 +4111,44 @@ private void createHiveTable(Database db, CreateTableStmt stmt) throws DdlExcept
         LOG.info("successfully create table[{}-{}]", tableName, tableId);
     }
 
+    private void createHudiTable(Database db, CreateTableStmt stmt) throws DdlException {
+        String tableName = stmt.getTableName();
+        List<Column> columns = stmt.getColumns();
+        long tableId = getNextId();
+        HudiTable hudiTable = new HudiTable(tableId, tableName, columns, stmt.getProperties());
```

Comment:
Where to get the table schema if no columns are specified when creating the table？

Reply:
do not get table schema when create table, it will get schema when query hudi table. 
please refer https://github.com/apache/incubator-doris/pull/9752